AirCasting/SearchAndFollow/SearchAndFollowMap/SearchAndFollowMap.swift

```diff
@@ -111,13 +123,15 @@ struct SearchAndFollowMap: UIViewRepresentable {
         }
         
         func mapView(_ mapView: GMSMapView, didTap marker: GMSMarker) -> Bool {
-            // Will be implemented in next task
+            guard let newPointerID = marker.userData as? Int else {
+                Log.error("To continue the program ensure yourself why newPointerID is not and INT!")
```

Comment:
What do you think about changing the message to "Unexpectedly found value other than Int in marker userData"?